app/models/authority.rb

```diff
@@ -261,7 +261,16 @@ def load_councillors(popolo)
 
     persons.map do |person|
       councillor = councillors.find_or_create_by(name: person.name)
-      councillor.update(email: person.email, image_url: person.image, party: person.party, popolo_id: person.id)
+
+      councillor.popolo_id = person.id
+
+      councillor.image_url = if councillor.cached_image_available?
+        councillor.cached_image_url
+      else
+        person.image
+      end
+
+      councillor.update(email: person.email, party: person.party)
```

Comment:
Yep, like the `image_url` being assigned. This is because the [#update method](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update) just calls [#save](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save) on the record, persisting all changed attributes.